app/models/claim_update.rb

```diff
@@ -23,6 +23,8 @@ class ClaimUpdate
 
   DEPENDENT_ELIGIBILITY_ANSWERS = {
     "claim_school_id" => "employment_status",
+    "subjects_taught" => "had_leadership_position",
```

Comment:
Are these two actually dependent?

Reply:
Nope. It was a relic of an intermediary step. It's gone now.